packages/react-components/src/DocsComponents/VersionSelector.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import {
+  MenuButton,
+  Menu,
+  MenuPopover,
+  MenuItemRadio,
+  MenuList,
+  MenuTrigger,
+  makeStyles,
+  MenuProps,
+  MenuItem,
+} from '../index';
+import { version as packageJsonVersion } from '../../package.json';
+import * as semver from 'semver';
+
+const useStyles = makeStyles({
+  menuButton: {
+    minWidth: '210px',
+    justifyContent: 'flex-start',
+    marginLeft: '5px',
+  },
+  chevronIcon: {
+    marginLeft: 'auto',
+  },
+  menuPopover: {
+    minWidth: '210px',
+    zIndex: 1000,
+  },
+  menuList: {
+    overflowY: 'auto',
+    maxHeight: '30vh',
+  },
+  menuItemRadio: {
+    height: 'auto',
+    paddingTop: '7px',
+    paddingBottom: '7px',
+  },
+});
+
+const onCheckedValueChange: MenuProps['onCheckedValueChange'] = (e, data) => {
+  const selectedUrl = data.checkedItems[0] + (window.top || window).location.search;
+  (window.top || window).location.href = selectedUrl;
+};
+
+/**
+ * Theme picker used in the react-components docs header
+ */
+export const VersionSelector: React.FC = () => {
+  const styles = useStyles();
+  const [versions, setVersions] = React.useState<string[][]>([]);
+
+  React.useEffect(() => {
+    // metadata.json is populated only in Chromatic
+    // an example of the file is available here:
+    // https://master--5ccbc373887ca40020446347.chromatic.com/metadata.json
+    fetch('/metadata.json').then(async response => {
+      if (response.ok) {
+        const versionsCleaned: string[][] = [];
+        const tagUrlMap: { [key: string]: string } = (await response.json()).versions;
+
+        Object.entries(tagUrlMap).forEach(([gitTag, url]) => {
+          const versionDirty = gitTag.split('_').pop() ?? '';
```

Comment:
@PeterDraex  please add a comment why/how we are clearning the version. I think the fallback should just be to render the bad version in full instead of omitting it

Reply:
added